This check is broken on Bionic because of the config file provided by the older libpugixml package.
Index: carta-backend-beta-2.0+2021.4.6/CMakeLists.txt
===================================================================
--- carta-backend-beta-2.0+2021.4.6.orig/CMakeLists.txt
+++ carta-backend-beta-2.0+2021.4.6/CMakeLists.txt
@@ -45,7 +45,7 @@ endif ()
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 FIND_PACKAGE(CURL REQUIRED)
-FIND_PACKAGE(PUGIXML REQUIRED)
+#FIND_PACKAGE(PUGIXML REQUIRED)
 FIND_PACKAGE(ZFP CONFIG REQUIRED)
 FIND_PACKAGE(PkgConfig REQUIRED)
 
